CRM/Utils/Token.php

```diff
@@ -732,22 +732,12 @@ public static function getContactTokenReplacement(
       $value = "cs={$cs}";
     }
     else {
-      $value = CRM_Utils_Array::retrieveValueRecursive($contact, $token);
+      $value = (array) CRM_Utils_Array::retrieveValueRecursive($contact, $token);
 
-      // FIXME: for some pseudoconstants we get array ( 0 => id, 1 => label )
-      if (is_array($value)) {
-        $value = $value[1];
```

Comment:
This line causes an enotice without this cleanup - it was added as part of a fix for https://issues.civicrm.org/jira/browse/CRM-13161 - however, all those greetings and gender fields are now tested & don't appear to have regressed (based on the tests I ran locally) with this change. 

The enotice directly mapped to the token processor having multi-value custom field data